metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/interfaces/MessageParser.java

```diff
@@ -34,15 +34,15 @@
    * @param rawMessage
    * @return If null is returned, this is treated as an empty list.
    */
-  List<T> parse(byte[] rawMessage);
+  List<T> parse(byte[] rawMessage, SensorParserConfig sensorParserConfig);
```

Comment:
Before we go any further I think we need to decide if changing the MessageParser interface is acceptable or not.  If I remember correctly the change to the interface is what prompted this whole conversation.

My opinion is that we quit worrying so much about the interface changing and instead focus on getting it right.  Thoughts?